### PR TITLE
Completed Code Review

### PR DESCRIPTION
**Pull request by DMattDraper**
1. Reviewed code base, searching for code smells and problems
   - Tracked the smells/problems in CODE_REVIEW.md
   - Fixed one of the problems mentioned by creating a distinct file for book state
2. Fixed accessibility issues by addressing the issues found in lighthouse reports
3. Implemented failure case functionality in the reading list reducer as a result of test failures

Lint:
image

Test:
image

e2e:
image

### CODE REVIEW

**Comment by DMattDraper** [comment]
Lighthouse Report:
image

File: angular.json [hunk 1]
```diff
@@ -339,15 +375,21 @@
...
+            ]
           }
         }
       }
     }
   },
   "cli": {
-    "defaultCollection": "@nrwl/angular"
+    "defaultCollection": "@nrwl/angular",
+    "analytics": false
```
Comment (kandarp-patel51) [inline angular.json:392]: What is the use of this?

File: CODE_REVIEW.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Code Review #
```
Comment (kandarp-patel51) [inline CODE_REVIEW.md:1]: Add more code smells and accessibility issue as asked in task.

File: angular.json (hunk 1 above)
Comment (DMattDraper) [reply angular.json:392]: This was not a manual change I made myself, I believe this changed when I ran npm install on the project

Comment (kandarp-patel51) [reply angular.json:392]: We should not have such kind of changes in PR.

File: libs/books/data-access/src/lib/+state/books.state.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    loaded: boolean;
+    error?: string | null;
+    searchTerm?: string;
+}
+
+export interface BooksPartialState {
+    readonly [BOOKS_FEATURE_KEY]: State;
+  }
+  
+  export const booksAdapter: EntityAdapter<Book> = createEntityAdapter<Book>();
```
Comment (kandarp-patel51) [inline libs/books/data-access/src/lib/+state/books.state.ts:16]: Align these two statements properly

**Pull request closed without merging**